.travis.yml

```diff
@@ -58,26 +58,20 @@ jobs:
     ## Define the rest of the matrix based on Kitchen testing
     # Make sure the instances listed below match up with
     # the `platforms` defined in `kitchen.yml`
-    # NOTE: Please try to select up to six instances that add some meaningful
-    #       testing of the formula's behaviour. If possible, try to refrain from
-    #       the classical "chosing all the instances because I want to test on
-    #       another/all distro/s" trap: it will just add time to the testing (see
-    #       the discussion on #121).  As an example, the set chosen below covers
-    #       the most used distros families, systemd and non-systemd and the latest
-    #       three supported Saltstack versions with python2 and 3.
-    #       As for `kitchen.yml`, that should still contain all of the platforms,
-    #       to allow for comprehensive local testing
-    #       Ref: https://github.com/saltstack-formulas/template-formula/issues/118
-    #       Ref: https://github.com/saltstack-formulas/template-formula/issues/121
-    - env: INSTANCE=default-ubuntu-1804-master-py3
+    - env: INSTANCE=default-ubuntu-2004-master-py3
+    # - env: INSTANCE=default-ubuntu-1804-master-py3
     # - env: INSTANCE=default-centos-8-master-py3
-    # - env: INSTANCE=default-ubuntu-1804-3000-3-py3
+    # - env: INSTANCE=default-ubuntu-2004-3001-py3
+    # - env: INSTANCE=default-ubuntu-1804-3001-py3
     - env: INSTANCE=default-centos-8-3001-py3
     # - env: INSTANCE=default-centos-7-3001-py3
-    # - env: INSTANCE=default-ubuntu-1804-3001-py3
+    - env: INSTANCE=default-ubuntu-1804-3000-3-py3
     # - env: INSTANCE=default-centos-8-3000-3-py3
     # - env: INSTANCE=default-centos-7-3000-3-py3
+    # - env: INSTANCE=default-ubuntu-1804-3000-3-py2
+    - env: INSTANCE=default-ubuntu-1604-3000-3-py2
     # - env: INSTANCE=default-ubuntu-1804-2019-2-py3
+    # - env: INSTANCE=default-ubuntu-1604-2019-2-py3
     # - env: INSTANCE=default-centos-8-2019-2-py3
     - env: INSTANCE=default-centos-7-2019-2-py3
```

Comment:
All of these entries actually work:

* https://travis-ci.org/github/myii/suricata-formula/builds/701828653

Selected the balance of platform, Salt version and Python version.

Reply:
Awesome. I'm surprised the ubuntu 16.04 works but I guess I'll take it. Thank You